### DISCUSSION TITLE
Category: categorizing photos on a daily basis

### DISCUSSION THREAD

**Discussion by xqyi** [photoprism/photoprism#4029] [Q&A]
When categorizing photos in the calendar based on time, it can only be done by year or month. In reality, on a single day, there could be nearly a thousand photos, and categorizing them only by month still results in a large number of photos.
I hope that when categorizing by time, it can be further refined to categorize photos on a daily basis.

**Comment by graciousgrey** [comment]
If you want to see photos of a certain day you can use the search for this. e.g `day:5 month:12 year:2023`

See also https://docs.photoprism.app/user-guide/search/filters/

**Comment by xqyi** [comment]
thanks. this has resolved my confusion very well.
